vmm/src/vm.rs

```diff
@@ -1658,7 +1661,7 @@ impl Vm {
 
     /// Gets the actual size of the balloon.
     pub fn get_balloon_actual(&self) -> u64 {
```

Comment:
I think this should change to `balloon_size()` to be consistent.

Reply:
done!